scripts/run.js

```diff
@@ -1,21 +1,26 @@
-const path = require('path');
+#!/usr/bin/env -S node --no-warnings=ExperimentalWarning
```

Comment:
This line is useless, unless `run.js` is called as an executable, which is not the case in our setup, especially when called from `npm run`, where we specify explicitly `node scripts/run.js`. I think we should remove this line.